projects/storefrontlib/src/cms-components/user/logout/logout-guard.ts

```diff
@@ -30,14 +52,20 @@ export class LogoutGuard implements CanActivate {
         type: PageType.CONTENT_PAGE,
       })
       .pipe(
-        tap(hasPage => {
-          if (!hasPage) {
-            this.routing.go({ cxRoute: 'home' });
-          }
-        })
+        filter(hasPage => !hasPage),
+        tap(() => this.redirect())
       );
   }
 
+  protected redirect(): void {
+    const cxRoute =
+      this.protectedRoutes && this.protectedRoutes.isAppProtected()
```

Comment:
Redirecting to a different page than before (to login instead of home - even if homepage is not protected) is a functional breaking change (it may surprise customers). To avoid that, let's also check if the feature level is `1.4`.